src/Apps/Artist/Routes/Overview/Components/ArtistRecommendations.tsx

```diff
@@ -0,0 +1,57 @@
+import { Serif } from "@artsy/palette"
+import { ArtistRecommendations_artist } from "__generated__/ArtistRecommendations_artist.graphql"
+import { ArtistRecommendationsQuery } from "__generated__/ArtistRecommendationsQuery.graphql"
+import { Overview_artist } from "__generated__/Overview_artist.graphql"
+import { SystemContext } from "Artsy"
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+import React, { useContext } from "react"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+
+interface ArtistRecommendationsProps {
+  artist: ArtistRecommendations_artist
+}
+
+export const ArtistRecommendations: React.FC<
+  ArtistRecommendationsProps
+> = props => {
+  const {
+    artist: { name },
+  } = props
+
+  return (
+    <div>
+      <Serif size="8" color="black100">
+        Related to {name}
+      </Serif>
+    </div>
+  )
+}
+
+export const ArtistRecommendationsFragmentContainer = createFragmentContainer(
+  ArtistRecommendations,
+  graphql`
+    fragment ArtistRecommendations_artist on Artist {
+      name
+    }
+  `
+)
+
+export const ArtistRecommendationsQueryRenderer: React.FC<{
+  artist: Overview_artist
```

Comment:
It might be better not to pass the whole artist object here, but just `artistID`. Then the invocation is `<ArtistRecommendations artistID=...`, which might make it easier to reuse this component outside the overview tab.

Although, that might be a premature optimization since I don't believe there's any plans for that, so feel free to disregard.

Reply:
I did feel a little weird about pulling in an object from a different component (`Overview_artist`)...so I'll make this change.

Reply:
Done!